### ISSUE TITLE
mouse clicking confusion

### ISSUE THREAD

**Issue by tbeason** [pop-os/pop#896]
**Distribution (run `cat /etc/os-release`):**
 
 NAME="Pop!_OS"
VERSION="19.10"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.10"
VERSION_ID="19.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=eoan
UBUNTU_CODENAME=eoan
LOGO=distributor-logo-pop-os
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**

Frequently, instead of left clicking, my mouse will right click. It happens regardless of which programs I am working in or have open. Holding down and dragging to select text takes several attempts because the text will not stay selected after releasing the mouse button. 

**Steps to reproduce (if you know):**

**Expected behavior:**

Left click when I left click, right click when I right click!

**Other Notes:**

I am on a desktop with a plain Logitech mouse. I have been using this setup (19.10 + same hardware) for months without issue. I'd say the problems started within the last ~6 weeks. The only updates I have been applying are the updates sent to me via the Pop! Shop. Rebooting does not fix the issue.

**Comment by joshua-noronha13** [comment]
This issue might not be a bug, please check settings > devices > mouse and touchpad.
There check whether your primary button is selected on left. Select left if it isn't already left.

This should be the required setting:
![Required Setting](https://user-images.githubusercontent.com/26717735/78970101-2badb700-7b26-11ea-8c12-a3cdd9df0aa5.png)

**Comment by tbeason** [comment]
No that's not it (although I did go verify that the setting is correct).

The behavior is such that (guessing):
- 50% of the time I correctly get a single left click
- 30% of the time I get a right click instead of a left click
- 20% of the time I get a sort of "combined" click (which I think is a left followed immediately by a right)

Note that it is actually possible to distinguish between cases 2 and 3. For example, single right clicking on a Firefox tab does not actually open that tab, it just opens the menu. A left followed by a right displays the tab and the menu.

I did try switching the USB port that the mouse was plugged into, no change in behavior.

To eliminate the suspicion that it is my mouse, I would just buy a new mouse. But even if I one bought today it wouldn't get here for weeks!
